Analysis/PIDTools/src/CreatePDFProcessor.cc

```diff
@@ -489,7 +537,7 @@ void CreatePDFProcessor::end() {
   _fpdf = new TFile(_filename.c_str(), "RECREATE");
```

Comment:
Memory leak here with _fpdf, but also with _myPID that should both deleted in the `end()` method